### PR TITLE
[bnb] Move documentation

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

- move bnb documentation to `perf_infer_gpu_many.mdx` as it was previously set to `perf_train_gpu_one.mdx` which is not relevant in the case of `bitsandbytes` integration since it supports inference only.

cc @stas00

I do have a question though, what about `perf_infer_gpu_one.mdx`? I think that the `bnb` documentation could fit well in this file as well since it supports single GPU inference too.

### CODE REVIEW

**Comment by stas00** [comment]
yeah, I wasn't sure, probably you're right and then like  perf_train_gpu_many.mdx says on top to first read perf_train_gpu_one.mdx - add the same to perf_infer_gpu_many.mdx?

**Comment by younesbelkada** [comment]
Yep makes sense! I propose a small refactoring at 2018285dd60c34d2654b0c51e523d7b1e815b989 !
Let me know if this works for you

File: docs/source/en/perf_infer_gpu_many.mdx [hunk 1]
```diff
@@ -11,4 +11,43 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 
 # Efficient Inference on a Multiple GPUs
 
-This document will be completed soon with information on how to infer on a multiple GPUs. In the meantime you can check out [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
\ No newline at end of file
+This document contains information on how to efficiently infer on a multiple GPUs. In the meantime you can check out [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
```
Comment (stas00) [inline docs/source/en/perf_infer_gpu_many.mdx:14]:
```suggestion
This document contains information on how to efficiently infer on a multiple GPUs. 

Make sure to first read on the [inference on a single GPU](perf_infer_cpu) as most of the techniques listed there would benefit the multi-gpu set up as well.
```

Comment (stas00) [reply docs/source/en/perf_infer_gpu_many.mdx:14]: do you think we should link to the CPU as well? I wasn't sure of the relevancy.

**Comment by stas00** [comment]
This would be difficult to maintain and 2 copies will get out of sync. Only one copy please - if you prefer one gpu that's where it should be.

**Comment by younesbelkada** [comment]
Proposed a change in 5f8a3aeb035e8b086d0f9045550000b6be0fb630 ! Let me know if this works for you

File: docs/source/en/perf_infer_gpu_many.mdx [hunk 2]
```diff
@@ -11,4 +11,16 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 
-This document will be completed soon with information on how to infer on a multiple GPUs. In the meantime you can check out [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
\ No newline at end of file
+This document contains information on how to efficiently infer on a multiple GPUs. 
+
+Make sure to first read on the [inference on a single GPU](perf_infer_cpu) as most of the techniques listed there would benefit the multi-gpu set up as well.
+
+<Tip>
+
+ Note: A multi GPU setup can use the majority of the strategies described in the [single GPU section](perf infer gpu one). You must be aware of simple techniques, though, that can be used for a better usage.
```
Comment (stas00) [inline docs/source/en/perf_infer_gpu_many.mdx:20]:
```suggestion
Note: A multi GPU setup can use the majority of the strategies described in the [single GPU section](perf infer gpu one). You must be aware of simple techniques, though, that can be used for a better usage.
```

File: docs/source/en/perf_infer_gpu_many.mdx [hunk 3]
```diff
@@ -11,4 +11,16 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 
 # Efficient Inference on a Multiple GPUs
 
-This document will be completed soon with information on how to infer on a multiple GPUs. In the meantime you can check out [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
\ No newline at end of file
+This document contains information on how to efficiently infer on a multiple GPUs. 
+
+Make sure to first read on the [inference on a single GPU](perf_infer_cpu) as most of the techniques listed there would benefit the multi-gpu set up as well.
+
```
Comment (stas00) [inline docs/source/en/perf_infer_gpu_many.mdx:17]:
```suggestion
```

File: docs/source/en/perf_infer_gpu_many.mdx [hunk 4]
```diff
@@ -11,4 +11,16 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+<Tip>
+
+ Note: A multi GPU setup can use the majority of the strategies described in the [single GPU section](perf infer gpu one). You must be aware of simple techniques, though, that can be used for a better usage.
+
+</Tip>
+
+## `bitsandbytes` integration for Int8 mixed-precision matrix decomposition - multi GPU setup
+
+Note that this features is also totally applicable in a single GPU setup as well - Check out the [single GPU section](perf_infer_gpu_one) for more details about the usage of it.
```
Comment (stas00) [inline docs/source/en/perf_infer_gpu_many.mdx:26]:
```suggestion
```

Currently, we don't repeat the entries at all - we just point the whole doc to read first.

This is not how it used to be, but that was chosen to be the new way, so I'm just following through.

**Review by stas00 — APPROVED** [review]
with a few tweaks I proposed this will now look good to merge.

thank you, @younesbelkada for working on it!

File: docs/source/en/perf_infer_gpu_many.mdx (hunk 3 above)
Comment (stas00) [reply docs/source/en/perf_infer_gpu_many.mdx:17]: remove repetition.

**Comment by younesbelkada** [comment]
Thanks a lot @stas00 for iterating on the change 💪

**Pull request merged**